requirements.txt

```
@@ -20,7 +20,7 @@ Flask-Compress==1.15
 Flask-RESTful==0.3.10
 fprime-fpl-layout==1.0.3
 fprime-fpl-write-pic==1.0.3
-fprime-fpp==3.1.0a7
+fprime-fpp==v3.1.0a8
```

Both work but for consistency...
```suggestion
fprime-fpp==3.1.0a8
```